setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='pipe-vms-research',
-    version='4.0.0',
+    version='4.1.0',
```

Comment:
Usually we do this the other way. We store the package metadata on package code (for example, and `__init__.py` file on module root) so that you can use it via `from regions import VERSION` wherever you need it in the package code, but also in `setup.py`. That way you don't need to read a file, parse, search via regex and so on.

Reply:
@andres-arana But this was done this other way, I only updated the patch. I also looked at the `pipe-research` v4 https://github.com/GlobalFishingWatch/pipe-research/blob/release-v4/setup.py and it's fixed there, actually [it was changed ](https://github.com/GlobalFishingWatch/pipe-research/compare/master...release-v4#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7)since in `master` seems that we were doing it as you suggested. Shouldn't `release-v4` had the latest approach on this?

Reply:
@rdgfuentes oh, I don't understand why we went forward with that approach on `release-v4`. Ok, let's keep it in the way you implemented it and I'll ask Matias once he's back.